new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -428,15 +430,53 @@ public static boolean deletePod(String name, String namespace) {
     return true;
   }
 
+  /**
+   * Returns the V1Pod object given the following parameters.
+   *
+   * @param namespace     in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
+   * @param podName       name of the pod to return
+   * @return V1Pod object if found otherwise null
+   */
+  public static V1Pod getPod(String namespace, String labelSelector, String podName) {
+    V1PodList pods = listPods(namespace, labelSelector);
+    for (var pod : pods.getItems()) {
+      if (podName.equals(pod.getMetadata().getName())) {
+        return pod;
+      }
+    }
+    return null;
+  }
+
+  /**
+   * Get the creationTimestamp for a given pod with following parameters.
+   *
+   * @param namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
+   * @param podName  name of the pod
+   * @return creationTimestamp from metadata section of the Pod
+   */
+  public static String getPodCreationTimestamp(String namespace, String labelSelector, String podName) {
```

Comment:
Are there any reason you are returning a String instead of DateTime ?

Reply:
It is easy to compare the timestamp after I convert it to Long